### PR TITLE
chore(InputGroup): add plain button example

### LINKED ISSUE #3045: Add input group examples with plain button modifier
Core issue - https://github.com/patternfly/patternfly-next/issues/2299

Just need to add an example of the input group component with a `<Button variant="plain">` button.

### PR DESCRIPTION
**Pull request by nicolethoen**
**What**: Closes #3045

### CODE REVIEW

File: packages/react-core/src/components/InputGroup/examples/InputGroup.md [hunk 1]
```diff
@@ -58,6 +58,13 @@ class SimpleInputGroups extends React.Component {
             <SearchIcon />
           </Button>
         </InputGroup>
+        <br />
+        <InputGroup>
+          <TextInput name="textInput12" id="textInput12" type="search" aria-label="search input example" />
+          <Button variant="plain" aria-label="search button for search input">
```
Comment (mcoker) [inline packages/react-core/src/components/InputGroup/examples/InputGroup.md:64]: Can we change the `aria-label` to "popover for input"? The core example is wrong, too.

```suggestion
          <Button variant="plain" aria-label="Popover for input">
```

**Comment by mcoker** [comment]
@nicolethoen after looking at the full examples page, it looks like this should go under the "with popover" examples. Also this example and the existing popover example should likely be `type="text"`. WDYT?

**Comment by nicolethoen** [comment]
> @nicolethoen after looking at the full examples page, it looks like this should go under the "with popover" examples. Also this example and the existing popover example should likely be `type="text"`. WDYT?

@mcoker sounds good to me

**Comment by mcoker** [comment]
@nicolethoen thanks! Can you update the `type` on both of the popover inputs to be `type="text"` and update the `aria-label` on the plain button text input to match the other popover text input? Change `search input example` to `input example with popover`

**Pull request merged**